src/test/java/com/atlassian/db/replica/it/example/aurora/AuroraClusterTest.java

```diff
@@ -31,8 +33,10 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class AuroraClusterTest {
-    final String writerEndpoint = "jdbc:postgresql://database-1.cluster-crmnlihjxqlm.eu-central-1.rds.amazonaws.com:5432/newdb";
-    final String readerEndpoint = "jdbc:postgresql://database-1.cluster-ro-crmnlihjxqlm.eu-central-1.rds.amazonaws.com:5432/newdb";
+    final AuroraJdbcUrl writerUrl = new AuroraJdbcUrl(
+        AuroraEndpoint.parse("database-1.cluster-crmnlihjxqlm.eu-central-1.rds.amazonaws.com:5432"), "newdb");
```

Comment:
Can we avoid static methods in API?

Reply:
What's wrong with static methods in API?

Reply:
Nothing 😉.